targetome-numeric-2025.csl

```diff
@@ -0,0 +1,297 @@
+    <link href="http://www.zotero.org/styles/elsevier-vancouver" rel="template"/>
+    <link href="https://www.maxapress.com/targetome/for_authors" rel="documentation"/>
+    <author>
+      <name>Lucien Shaw</name>
+      <email>myxlc55@outlook.com</email>
+      <uri>https://github.com/lucienshawls</uri>
+    </author>
+    <category citation-format="numeric"/>
+    <summary>CSL citation style for Targetome (numeric, 2025), drafted based on the author guidelines at https://www.maxapress.com/targetome/for_authors and cross-checked against the official EndNote style file (https://www.maxapress.com/data/article/export-supplementary?id=3925). Note: the patent “Country” field in the EndNote template cannot be mapped directly in CSL, so the authority variable is used as a substitute.</summary>
+    <updated>2025-12-16T02:02:03+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author-names">
+    <names variable="author">
+      <name et-al-min="6" et-al-use-first="5" initialize-with="" name-as-sort-order="all"/>
+    </names>
+  </macro>
+  <macro name="editor-names">
+    <names variable="editor">
+      <name et-al-min="6" et-al-use-first="5" initialize-with="" name-as-sort-order="all"/>
+    </names>
+  </macro>
+  <macro name="issued-year">
+    <date date-parts="year" form="text" variable="issued"/>
+  </macro>
+  <macro name="event-year">
+    <date date-parts="year" form="text" variable="event-date"/>
+  </macro>
+  <macro name="event-place">
+    <text variable="event-place"/>
+  </macro>
+  <macro name="main-title">
+    <text variable="title" prefix=" "/>
+  </macro>
+  <macro name="container-title">
+    <text variable="container-title"/>
+  </macro>
+  <macro name="volume">
+    <text variable="volume"/>
+  </macro>
+  <macro name="issue">
+    <text variable="issue"/>
+  </macro>
+  <macro name="pages">
+    <text variable="page"/>
+  </macro>
+  <macro name="publisher-place">
+    <text variable="publisher-place"/>
+  </macro>
+  <macro name="publisher-name">
+    <text variable="publisher"/>
+  </macro>
+  <macro name="number-of-pages">
+    <text variable="number-of-pages"/>
+  </macro>
+  <macro name="edition">
+    <text variable="edition"/>
+  </macro>
+  <macro name="issued-date">
+    <date form="numeric" date-parts="year-month-day" variable="issued"/>
+  </macro>
+  <macro name="authority">
+    <text variable="authority" form="short"/>
+  </macro>
+  <macro name="number">
+    <text variable="number"/>
+  </macro>
+  <macro name="genre">
+    <text variable="genre"/>
+  </macro>
+  <macro name="url">
+    <text variable="URL"/>
+  </macro>
+  <macro name="entry-head">
+    <choose>
+      <if type="book" match="all" variable="editor">
+        <group suffix=",">
+          <text macro="editor-names"/>
```

Comment:
Calling a names variable and its label in two different places isn't really ideal. Please put them together.